test/test.ml

```diff
@@ -1,8 +1,6 @@
 type t
 
-external ava:< 
-    test : string -> (t -> unit) -> unit[@bs.meth]
-> Js.t = "ava" [@@bs.module]
+external ava:'a Js.t = "ava" [@@bs.module]
```

Comment:
This doesn't matter much for an internal binding, but technically this means `ava` can be inferred to be an `int Js.t`, so you should generally avoid using `'a` in place of `< .. >`.